model/src/main/kotlin/Package.kt

```diff
@@ -104,7 +104,8 @@ data class Package(
 
     /**
      * Processed VCS-related information about the [Package] in normalized form. The information is either derived from
-     * [vcs], guessed from additional data as fallback or empty.
+     * [vcs], guessed from additional data as fallback or empty. On top of that [PackageCuration]s may have been
```

Comment:
The commit message should fix https://github.com/oss-review-toolkit/ort/issues/4618.